src/main/java/frc/robot/subsystems/ClimbSubsystem/ClimbSubsytem.java

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems.ClimbSubsystem;
+
+import java.util.Map;
+
+import com.ctre.phoenix6.hardware.TalonFX;
+
+import edu.wpi.first.math.trajectory.TrapezoidProfile.Constraints;
+import edu.wpi.first.wpilibj.shuffleboard.BuiltInLayouts;
+import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
+import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardLayout;
+import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardTab;
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.controller.ProfiledPIDController;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants.IntakeConstants;
+
+/** Add your docs here. */
+public class ClimbSubsytem extends SubsystemBase {
+    private TalonFX m_leftClimbMotor = new TalonFX(ClimbConstants.LEFT_CLIMB_MOTOR_ID, "rio");
+    private TalonFX m_rightClimbMotor = new TalonFX(ClimbConstants.RIGHT_CLIMB_MOTOR_ID, "rio");
+    private Constraints constraints = new Constraints(ClimbConstants.MAX_VELOCITY, ClimbConstants.MAX_ACCELERATION);
+
+
+    private ProfiledPIDController leftpidController = new ProfiledPIDController(ClimbConstants.CLIMB_PID_KP, ClimbConstants.CLIMB_PID_KI, ClimbConstants.CLIMB_PID_KD, constraints);
+    private ProfiledPIDController rightpidController = new ProfiledPIDController(ClimbConstants.CLIMB_PID_KP, ClimbConstants.CLIMB_PID_KI, ClimbConstants.CLIMB_PID_KD, constraints);
+
+    public ClimbSubsytem() {
+        ShuffleboardTab tab = Shuffleboard.getTab("Testing");
+        ShuffleboardLayout climbCommandsLayout = tab.getLayout("TestingCommands", BuiltInLayouts.kList)
+            .withSize(2, 2)
+            .withProperties(Map.of("Label position", "HIDDEN"));
+
+        climbCommandsLayout.add(motorUp());
+        climbCommandsLayout.add(motorDown());
+
+}
+
+    @Override
+    public void periodic() {
+        double rightpidOutput = rightpidController.calculate(m_rightClimbMotor.getPosition().getValue());
+        double leftpidOutput = leftpidController.calculate(m_leftClimbMotor.getPosition().getValue());
```

Comment:
Should clamp the pid outputs to between -1 and 1.

https://github.com/gwhs/2024v2/pull/70/files#diff-a7320dd42369737b0eb7a20f9c356a9e8705501688f460003cd26a38482edaedR57